outlines/welcome.md

```diff
@@ -0,0 +1,31 @@
+1. Welcome to the Meteor guide
+  1. This is a set of articles outlining opinions on best-practice application development in the Meteor framework (and in many cases other frameworks)
+  2. Nothing here is *required* (i.e. you can use the platform in many other ways), but this is how we think apps are best developed. Take what you want from it but be aware that these are the community conventions
+  3. Hard APIs can be found at [docs] and community packages at [atmosphere].
+2. What is Meteor?
+  1. A set of JS client + server code for building web and mobile applications, built and curated by MDG and the Meteor community.
+  2. A build tool for building JS bundles for deployment
+3. Meteor principles
+  1. Data on the Wire. Meteor doesn't send HTML over the network. The server sends data and lets the client render it.
+  1. One Language. Meteor lets you write both the client and the server parts of your application in JavaScript.
+  1. Database Everywhere. You can use the same methods to access your database from the client or the server.
```

Comment:
Perhaps: "Database Everywhere. Query data on the client using the same syntax you use on the server."

I think the data writing case is encapsulated inside Optimistic UI?